src/couch/test/eunit/couch_db_split_tests.erl

```diff
@@ -16,6 +16,7 @@
 -include_lib("couch/include/couch_db.hrl").
 
 -define(RINGTOP, 2 bsl 31).
+-define(TIMEOUT, 60). % seconds
```

Comment:
Can you please leave the timeout bumps in a separate commit to the Jenkins setup / junit formatting stuff?

Reply:
Can do.